pom.xml

```diff
@@ -99,12 +100,32 @@
 			</dependency>
 		</dependencies>
 	</dependencyManagement>
-	<!--TODO: Move transport related dependencies here pending https://jira.spring.io/browse/XD-3337  -->
 	<dependencies>
 		<dependency>
 			<groupId>org.springframework.cloud</groupId>
 			<artifactId>spring-cloud-stream</artifactId>
 		</dependency>
+		<!-- Cloud connector dependencies -->
+		<!-- Lattice connector dependency to create services info from lattice -->
+		<dependency>
```

Comment:
I think it's pretty reasonable. A module developer would have the following options:

* new module developers (even outside the project) can extend from scsm-master, thus getting the dependency;
* developers that embed a module and add it as a dependency can explicitly include the dependencies;
* developers that embed a module may add it as parent and get the dependency automatically.